openshift_pipelines/installing-pipelines.adoc

```diff
@@ -0,0 +1,19 @@
+[id="installing-pipelines"]
+= Installing OpenShift Pipelines
```

Comment:
Not sure if it's relevant but should we add a upgrade section a well and just point out or link to how operators update automatically?

Reply:
Hi @siamaksade - In the Installing Pipelines section, I am pointing to the following OCP doc section  - https://installing-pipelines--ocpdocs.netlify.com/openshift-enterprise/latest/operators/olm-adding-operators-to-cluster.html#olm-adding-operators-to-a-cluster

This section explains in details on the Upgrade options available during install (Approval strategy). 

Is my understanding correct or should I be pointing to some other section.

Reply:
In the install procedure using web console, we have pointed out to select approval strategy as ?*automatic* for upgrades to be done automatically by OLM. If they select *manual* , then for each upgrade manual approval will be required.

Reply:
@siamaksade do we want to mention upgrade from community operator? If not, I think that current version is ok.

Reply:
@siamaksade ^^

Reply:
As discussed with @nikhil-thomas and @ppitonak - TP resets the history here, and we may provide upgrade paths/procedure from TP 1.1 (next release) onwards

Reply:
@siamaksade - I will raise a separate PR for the upgrade procedure.